### DISCUSSION TITLE
Native AOT

### DISCUSSION THREAD

**Discussion by williamlzw** [cefsharp/CefSharp#4414] [General]
Can cefsharp support nativeaot?

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related  questions.

> Native AOT applications come with a few fundamental limitations and compatibility issues. The key limitations include:
> - No C++/CLI

As per https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/#limitations-of-native-aot-deployment

`Native AOT` doesn't support `C++/CLI` which `CefSharp` is built with.
